src/main/java/core/basesyntax/strategy/impl/BalanceOperation.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+
+public class BalanceOperation implements OperationHandler {
+    @Override
+    public void handle(FruitTransaction transaction) {
+        Storage.addFruit(transaction.getFruit(), transaction.getQuantity());

```

Comment:
Directly accessing the `Storage` class here makes the `BalanceOperation` class tightly coupled with the `Storage`. Consider passing a reference to a storage object through the constructor to follow the Dependency Inversion Principle, part of the SOLID principles.